ui_testing/pages/order_page.py

```diff
@@ -521,3 +521,28 @@ def remove_article(self, testplans=''):
             self.base_selenium.click(element='general:confirmation_button')
         self.sleep_small()
 
+    def set_random_test_unit(self, test_units='', sub_order_index=''):
```

Comment:
why do you need to define new function?
you can use https://github.com/modeso-open-source/1lims-automation/blob/b278dfc150ae4a54a0fdb25d1ea39f33c1081341/ui_testing/pages/order_page.py#L372

Reply:
@ghada-37 because this method will not work because I need to use set_test _unit  to set specific index and the set_test_unit in this method will not do that  ( for testunit in test_units:
            self.set_test_unit(test_units=testunit) )

Reply:
I still found u can use it 
```
update_suborder(sub_order_index=0,test_units=[random_row['name']])

Reply:
Done